d3-map-of-world/Lessons/0030_index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+    <title>World Population</title>
+  </head>
+  <body>
+    <!-- D3 v5 script -->
+    <script type="text/javascript" src="./d3v5min.js"></script>
+
+    <!--
+    There are many different file formats that hold geospatial data that you
```

Comment:
Really great information here. How about the following to shorten this description a bit?:

```
There are many different file formats that hold geospatial data that you can use to create maps. We're only going to focus on two that are web-friendly: GeoJSON and TopoJSON. For now, understand that they hold information that describes the shape of regions, states, provinces, and countries, among other things.

D3 provides a script that gets the necessary information from those formats so you can create a map. A copy of this script is saved in the project folder and named `topojsonv1min.js`.
    
Create another `script` tag with the `type` attribute set to `text/javascript` and the `src` attribute set to the name of the TopoJSON script file.
```